src/vconsole/90-vconsole.rules.in

```diff
@@ -7,4 +7,4 @@
 
 # Each vtcon keeps its own state of fonts.
 #
-ACTION=="add", SUBSYSTEM=="vtconsole", KERNEL=="vtcon*", RUN+="@rootlibexecdir@/systemd-vconsole-setup"
+ACTION=="add", SUBSYSTEM=="vtconsole", KERNEL=="vtcon*", ATTR{name}!="*dummy device", RUN+="@rootlibexecdir@/systemd-vconsole-setup"
```

Comment:
hmm, so this is racy: by the time the process is started the dummy device might already have disappeared again from the kernel. The above is hence merely an opimization, but doesn't fix anything, does it?